packages/govern-console/src/containers/CreateDAO/CreateDAO.tsx

```diff
@@ -464,48 +293,102 @@ const NewDaoForm: React.FC<FormProps> = memo(
                   justifyContent: 'space-between',
                 }}
               >
-                <InputField
-                  label=""
-                  onInputChange={onChangeTokenName}
-                  value={tokenName.current}
-                  height="46px"
-                  width="200px"
-                  placeholder={"Your Token's Name?"}
-                  error={mainInputsErrors.tokenNameError !== ''}
-                  helperText={mainInputsErrors.tokenNameError}
+                <Controller
+                  name="tokenName"
+                  control={control}
+                  defaultValue=""
+                  rules={{
+                    validate: (value) => {
+                      return !getValues('isExistingToken')
```

Comment:
@Rekard0 This validation function gets used multiple times.. We should create a function in the component and that's what should be used in all places.